python/lsst/meas/algorithms/gaussianPsfFactory.py

```diff
@@ -154,7 +167,10 @@ def makeField(cls, doc):
         def applyWrapper(config, **kwargs):
             """Construct a Gaussian PSF
 
-            @param[in] config: an instance of GaussianPsfFactory
+            Parameters
+            ----------
+            config : ``lsst.pex.config.ConfigurableField``
```

Comment:
The type here is `GaussianPsfFactory` (this class). I can see the confusion: this is a closure that is used by `ConfigurableField` to construct the correct kind of PSF, and so we're not documenting the return of `makeField` here. Keep the docstring as-written (even though it's sort of redundant).